README.md

```diff
@@ -2,7 +2,14 @@
 
 > Modern CSS linter.
 
-Supports the latest CSS syntax and features, including custom properties, range context for media features, calc() and nesting.
+## Features
+
+* _Nearly a hundred rules_ - from stylistic rules, such as checking the spacing around the colon in declarations, to rules that catch subtle coding mistakes, such as invalid hex colors or [overriding shorthand properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties#Tricky_edge_cases).
+* _Support for the latest CSS syntax_ - including custom properties, range context for media features, calc() and nesting.
+* _Completely unopinionated_ - only enable the rules you want and configure them with options to tailor the linter to your needs.
+* _Sharable configs_ - if you don't want to craft your own config you can use a shareable config, for example one that checks against the [SuitCSS code style](https://github.com/suitcss/suit/blob/master/doc/STYLE.md).
```

Comment:
I think we could cut this down to: "if you don't want to craft your own config you can use a shareable config."